scout/server/blueprints/cases/templates/cases/gene_panel.html

```diff
@@ -77,70 +78,70 @@
-              <input type="checkbox" class="ios8-switch" id="hpo_clinical_filter" onChange="this.form.submit()" name="hpo_clinical_filter"{% if case.hpo_clinical_filter %}checked{% endif %}>
-              <label for="hpo_clinical_filter"></label>
-            </form>
-          </div>
-          <div class="col-5 align-items-start" data-placement="top" data-toggle='tooltip'
-            title="Toggle on to use dynamic (HPO) panel instead of default gene panel for searching with clinical filter.">
-            Use HPO list for clinical filter
+      <div>
+        <div class="mt-3 d-flex justify-content-center">
+          <div class="col-7">
+            <div class="row">
+              <form action="{{ url_for('cases.update_clinical_filter_hpo', institute_id=institute._id, case_name=case.display_name)+'#hpo_clinical_filter' }}" method="POST">
+                <input type="checkbox" class="ios8-switch" id="hpo_clinical_filter" onChange="this.form.submit()" name="hpo_clinical_filter"{% if case.hpo_clinical_filter %}checked{% endif %}>
+                <label for="hpo_clinical_filter"></label>
+              </form>
+              <p data-placement="top" data-toggle='tooltip'
+                title="Toggle on to use dynamic (HPO) panel instead of default gene panel for searching with clinical filter.">&nbsp;&nbsp;Use HPO list for clinical filter</p>
+            </div>
+            <div class="row">
+              {% if case.track == 'cancer' %}
+                <a class="btn btn-secondary btn-sm text-white"
+                  href="{{ url_for('variants.cancer_variants', institute_id=institute._id,
+                                   case_name=case.display_name, variant_type='clinical',
+                                   gene_panels=['hpo']) }}">
+                   Clinical HPO SNV variants
+                 </a>
+              {% else %}
+                <a class="btn btn-secondary btn-sm text-white"
+                  href="{{ url_for('variants.variants', institute_id=institute._id,
+                                   case_name=case.display_name, variant_type='clinical',
+                                   gene_panels=['hpo']) }}">
+                   Clinical HPO SNV variants
+                 </a>
+               {% endif %}
+            </div>
           </div>
-          <div class="col-1">
-            <a href="{{ url_for('cases.download_hpo_genes', institute_id=institute._id, case_name=case.display_name, category="clinical") }}" class="btn btn-primary btn-xs var_btn"><span class="fa fa-download text-white" aria-hidden="true"></span></a>
+          <div class="col-5">
+            <div class="row">
+              Download HPO gene panel &nbsp;&nbsp; <a href="{{ url_for('cases.download_hpo_genes', institute_id=institute._id, case_name=case.display_name, category="clinical") }}" class="btn btn-primary btn-xs var_btn"><span class="fa fa-download text-white" aria-hidden="true"></span></a>
```

Comment:
i noticed that on the laptop screen, the (Download HPO gene panel) icon is in the next line
<img width="639" alt="Screen Shot 2021-06-17 at 16 47 21" src="https://user-images.githubusercontent.com/41540288/122420277-d96af000-cf8b-11eb-9aee-74871c7fd6bd.png">

Reply:
I've changed it like this now:
![image](https://user-images.githubusercontent.com/28093618/122512440-23de8200-d009-11eb-87f3-ce23668adfe6.png)

Reply:
Elegant 🏅